response.go

```diff
@@ -130,6 +130,41 @@ func (r *Response) Image(im image.Image) error {
 	return nil
 }
 
+// Attachment send image, audio, video and file attachment
+func (r *Response) Attachment(dataType, url string) error {
```

Comment:
We should probably get rid of the magic values introduce by the `dataType` string.

Something like the following would be great:

```go
type AttachmentType string

const (
   ImageAttachment AttachmentType = "image"
   AudioAttachment AttachmentType = "audio"
   VideoAttachment AttachmentType = "video"
   FileAttachment AttachmentType = "file"
)
```

So we can then change this function's signature to `func(to Recipient, dataType AttachmentType, url string) error`